runtime/frequency-rococo/Cargo.toml

```diff
@@ -21,6 +21,7 @@ log = { version = "0.4.17", default-features = false }
 scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
 serde = { version = "1.0.137", optional = true, features = ["derive"] }
 smallvec = "1.6.1"
+getrandom = { version = "0.2", default-features = false, features = ["js"] }
```

Comment:
I'm curious which pallet or part of runtime code needs randomness?

Reply:
I am too I just copied frequency to rococo but will find out from Shannon